### DISCUSSION TITLE
Show latest snapshot while loading camera view

### DISCUSSION THREAD

**Discussion by bubba925** [blakeblackshear/frigate#11860] [Ideas]
Currently on the 14 beta 2, when loading a camera view on a mobile connection or spotty wifi it can take some time and the current loading screen is just black. Would prefer the latest snapshot image or something  with a loading wheel to know its doing something or see what I need to see in the snapshot and not even have to load the view.  This is the current view.
![image](https://github.com/blakeblackshear/frigate/assets/118318345/2df7b4c7-c5d3-4b30-9aa6-476052b36965)

Might be completely unrelated but it also seems like when using Frigate proxy in HA remotely, load times seem much longer.

**Comment by hawkeye217** [comment]
Thanks for this. I've added a loading indicator in this PR: https://github.com/blakeblackshear/frigate/pull/11855
